src/on-premises/config/env/platform_back.md

```diff
@@ -0,0 +1,86 @@
+
+## RabbitMQ configuration
+| Variables         | Default value | Description              |
+|-------------------|---------------|--------------------------|
+| RABBITMQ_HOST     | "127.0.0.1"   | RabbitMQ server address. |
+| RABBITMQ_PORT     | "5672"        | RabbitMQ server port.    |
+| RABBITMQ_USER     | "bimdata"     | RabbitMQ username.       |
+| RABBITMQ_PASSWORD | "bimdata"     | RabbitMQ password.       |
+
+
+## OpenID configuration
+::: v-pre
+| Variables                    | Default value           | Description            |
+|------------------------------|-------------------------|------------------------|
+| OIDC_OP_ISSUER               | "http://localhost:8000" | OIDC provider address. |
+:::
+
+## Email configuration
+| Variables            | Default value          | Description                   |
+|----------------------|------------------------|-------------------------------|
+| SMTP_HOST            | "smtp.mandrillapp.com" | SMTP server address.          |
+| SMTP_PORT            | 587                    | SMTP server port.             |
+| SMTP_USE_TLS         | "True"                 | SMTP communication use TLS.   |
+| SMTP_USER            | "BIMData.io"           | SMTP authentication user.     |
+| SMTP_PASS            | False                  | SMTP authentication password. |
+| DEFAULT_FROM_EMAIL   | "support@bimdata.io"   | SMTP default from email.      |
+
+### Image configuration
+| Variables            | Default value | Description                                            |
+|----------------------|---------------|--------------------------------------------------------|
+| WORKERS              | 4             | Configure Gunicorn workers.                            |
+| PORT                 | 8000          | Configure Gunicorn listen port.                        |
+| CA_CERT              | ""            | Path of a certificate to add to container trusted CAs. |
+| COMPILE_SCSS         | 0             | 0 or 1. Configure if django compilescss during init.   |
+| COLLECT_STATIC       | 1             | 0 or 1. Configure if django collectstatic during init. |
+| APPLY_MIGRATION      | 1             | 0 or 1. Configure if django migrate during init.       |
+| PROCESS_TASKS        | 0             | 0 or 1. Configure if django process_tasks.             |
+
+## Other configuration
+| Variables                        | Default value                                      | Description |
+|----------------------------------|----------------------------------------------------|-------------|
+| ENV                              | "development"                                      |             |
+| ALLOWED_HOSTS                    |                                                    |             |
+| ADMIN_INTERFACE                  | True                                               |             |
+| SECRET_KEY                       | 7rvr*q1&_eqcetu^2x#2q+4&g8(&n&6*68+6xd#mxqs^6-u2rp |             |
```

Comment:
a default secret is a security issue. @amineau we should force a defined secret_key value when we're not in a development env